rules-reviewed/jboss-eap/uncategorized/environment-dependent.windup.xml

```diff
@@ -24,7 +23,7 @@
                 </javaclass>
             </when>
             <perform>
-                 <hint title="Dynamic instantiation of class" effort="0" severity="optional">
+                 <hint title="Dynamic instantiation of class" effort="1" severity="mandatory">
```

Comment:
I have added 1 extra story point for Class.forName() even though we didn't agree on it in the issue. I think it makes sense, or does it not?